app/commands/v2/put_content.rb

```diff
@@ -219,7 +225,7 @@ def locale
       end
 
       def publishing_app
-        payload.fetch(:publishing_app)
+        payload.fetch(:publishing_app, nil)
```

Comment:
This is equivalent to `payload[:publishing_app]` which is a bit clearer for this use case.